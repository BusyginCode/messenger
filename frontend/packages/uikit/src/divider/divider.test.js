import React from 'react'
import { shallow } from 'enzyme'
import Divider from './divider'

describe('uikit/Divider', () => {
  it('should render correctly', () => {
    expect(shallow(<Divider />)).toMatchSnapshot()
  })
})
