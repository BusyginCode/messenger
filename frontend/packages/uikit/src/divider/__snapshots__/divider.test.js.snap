// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`uikit/Divider should render correctly 1`] = `
<div
  className="divider default m"
  style={
    Object {
      "height": undefined,
    }
  }
/>
`;
